src/applications/claims-status/sass/claims-status.scss

```diff
@@ -556,6 +556,27 @@ h1:focus {
 .claim-timeline {
   padding-top: 0;
 
+  h5 button {
```

Comment:
@jbalboni @1Copenut This should cover the IE/Edge inconsistency, I am, however, having issues testing authenticated stuff via my Windows VM. Do either of you have a Windows machine handy to test the review instance? 

Also, should this live as a mixin - are we going to reuse on the Appeals timeline (or elsewhere)? 

If so, that should be changed/added to the `design-systems` repo correct?

Reply:
@kfrz I can test this, but it won't be until end of day Central Time. Will let you know then any issues or if things line out as expected on IE11 and Edge.

Reply:
I can check it in Browserstack, but it looks like the build failed, so I don't think the review instances is up to date yet.

I'm not sure this should be a mixin. I think we can wait on that until we come across more instances of it.

Also, I think you could change the selector here. Putting a class on the button and using that would be best (e.g. `.section-header-button` or something like that), but you could do `.section-header > button` if the single class one doesn't override everything you need.